uiTests/src/test/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala

```diff
@@ -2,18 +2,40 @@ package com.virtuslab.gitmachete.uitest
 
 import java.nio.file.Files
 import java.nio.file.attribute.FileTime
-
 import com.virtuslab.gitmachete.testcommon.BaseGitRepositoryBackedIntegrationTestSuite
 import com.virtuslab.gitmachete.testcommon.BaseGitRepositoryBackedIntegrationTestSuite.SETUP_WITH_SINGLE_REMOTE
-import org.junit.{After, Assert, Before, Test}
+import org.junit.{ After, AfterClass, Assert, Before, BeforeClass, Test }
 import org.junit.runner.RunWith
 import org.junit.runners.JUnit4
 import org.virtuslab.ideprobe.Extensions._
 import org.virtuslab.ideprobe._
 import org.virtuslab.ideprobe.config._
 import org.virtuslab.ideprobe.dependencies._
 import org.virtuslab.ideprobe.ide.intellij.IntelliJFactory
-import org.virtuslab.ideprobe.junit4.RunningIntelliJPerSuite
+
+trait RunningIntelliJPerSuite extends RunningIntelliJPerSuiteBase {
+  @BeforeClass override final def setup(): Unit = super.setup()
+
+  @AfterClass override final def teardown(): Unit = {
+    try {
+      super.teardown()
+    } catch {
+      case e: Exception =>
+        val filteredSuppressed = e.getSuppressed.filterNot { s =>
+          // A duct tape to ignore https://github.com/VirtusLab/ide-probe/issues/95
+          s.getMessage.contains("scala.MatchError: IDE_UPDATE (of class com.intellij.notification.NotificationType)") ||
```

Comment:
It doesn't look great but just as you pointed out, this is duct tape we are planning to remove when appropriate issue is closed?

Reply:
Yes, I hope so ;)